### PR TITLE
Add `BBC` to copyright text for all services

### LINKED ISSUE #2667: Add `BBC` to copyright text for all services
**Is your feature request related to a problem? Please describe.**
The word `BBC` doesn't appear in the copyright text for some services.
consider these two screenshots from `igbo` and `russian` service respectively. Note that one has the word `BBC` in the copyright text and the disclaimer while the other only has `BBC` in the disclaimer.
(_NB: I used google translate to display them in English_)

Screen Shot 2019-07-23 at 11 16 28 AM
Screen Shot 2019-07-23 at 11 16 19 AM

**Describe the solution you'd like**
There needs to be the `BBC` mentioned twice. The first mention is for the `BBC.` in `© 2019 BBC. ` and the second mention for the disclaimer sentence. e.g. for Russian it should be `'Би-би-си. Би-би-си не несет ответственности за содержание других сайтов.',`
This then shows up online as `© 2019 Би-би-си. Би-би-си не несет ответственности за содержание других сайтов.',` 

**Describe alternatives you've considered**

**Testing notes**
Since this is a translation fix, just a visual regression should be sufficient. We should separately have an issue to add automated visual regression checks, but that shouldn't be in this ticket.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
Raised from https://github.com/bbc/psammead/issues/934

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #2667 

**Overall change:** _Add the word `BBC` in its correct translation to the copyright text for all services._

**Code changes:**

_Add the word `BBC` in its correct translation to the copyright text in config files for all services._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/lib/config/services/pashto.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ const service = {
         text: 'Contact the BBC',
       },
     ],
-    copyrightText: 'بي بي سي د پردېو ویبپاڼو د محتوا مسوله نه ده.',
+    copyrightText: 'بی بی سی. بي بي سي د پردېو ویبپاڼو د محتوا مسوله نه ده.',
```
Comment (DenisHdz) [inline src/app/lib/config/services/pashto.js]:
```suggestion
    copyrightText: 'بی بی سی. بي بي سي د پردېو ویبپاڼو د محتوا مسوله نه ده',
```

File: src/app/lib/config/services/tigrinya.js [hunk 2]
```diff
@@ -155,7 +155,7 @@ const service = {
         text: 'Contact the BBC',
       },
     ],
-    copyrightText: 'ቢቢሲ ንትሕዝቶ ካልኦት ገጻት ኢንተርኔት ኣይሕተትን፡',
+    copyrightText: 'ቢቢሲ . ቢቢሲ ንትሕዝቶ ካልኦት ገጻት ኢንተርኔት ኣይሕተትን፡',
```
Comment (DenisHdz) [inline src/app/lib/config/services/tigrinya.js]:
```suggestion
    copyrightText: 'ቢቢሲ. ቢቢሲ ንትሕዝቶ ካልኦት ገጻት ኢንተርኔት ኣይሕተትን፡',
```

**Review by DenisHdz — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]
I dont speak these languages, but LGTM after denis's requested changes

**Review by sareh — APPROVED** [review]
👍 Looks great. I can only really read the Right-to-left services added and they look good. I checked the others that aren't `BBC.` in Google translate and they look good too. 
Screen Shot 2019-07-26 at 12 27 24
Screen Shot 2019-07-26 at 12 27 11
Screen Shot 2019-07-26 at 12 26 43

**Review by sareh — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Pull request merged**